test/jdk/java/lang/WrappersTest.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
We have historically not found it necessary to write tests to verify this kind of change and have instead relied on signature testing.

Reply:
+1. 

FWIW.  You have also verified when you get the warnings from JDK build and needs to suppressed it.

Reply:
The test can be removed but was a double check that all of the constructors of the named classes had been modified.